components/brave_wallet/browser/eth_data_builder.h

```diff
@@ -31,6 +31,19 @@ bool Allowance(const std::string& owner_address,
 
 }  // namespace erc20
 
+namespace erc721 {
+
+// Transfer ownership of an NFT.
+bool TransferFrom(const std::string& from,
```

Comment:
I hooked this one up first since it seems CyptoKitties is on an older standard which doesn't have safeTransferFrom